DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -653,10 +654,10 @@ public void GetEligibleCoursesToAddToGroup_does_not_return_courses_already_in_gr
                 .All()
                 .With(c => c.Active = true)
                 .Build();
-            var groupCourse = new GroupCourse { CustomisationId = 2 };
+            var groupCourse = new GroupCourse { CustomisationId = 2, Active = true, GroupId = 1 };

```

Comment:
Why do we need Active and GroupId?

Reply:
As well as the filter on group id, the group course must also be "active" - something that is checkd internally with the method.